src/python/pants/rules/core/strip_source_roots_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from unittest.mock import Mock
+
+from pants.build_graph.files import Files
+from pants.engine.fs import create_fs_rules
+from pants.engine.legacy.graph import HydratedTarget
+from pants.engine.rules import RootRule
+from pants.engine.selectors import Params
+from pants.rules.core.strip_source_root import SourceRootStrippedSources, strip_source_root
+from pants.source.source_root import SourceRootConfig
+from pants_test.subsystem.subsystem_util import init_subsystem
+from pants_test.test_base import TestBase
+
+
+class StripSourceRootsTests(TestBase):
+  @classmethod
+  def rules(cls):
+    return super().rules() + [
+      strip_source_root,
+      RootRule(SourceRootConfig),
+      RootRule(HydratedTarget),
+    ] + create_fs_rules()
+
+  def mock_hydrated_target(self, target_address, source_filename, type_alias=None):
+    adaptor = Mock()
+    adaptor.sources = Mock()
+    source_files = { source_filename: "print('random python')" }
```

Comment:
Python collection literals don’t pad spaces like this. Should be 

`{source_filename`...